app/assets/javascripts/helpers/toMomentFromRailsDate.js

```diff
@@ -21,3 +20,11 @@ export function toMomentFromRailsDate(railsDateString) {
 
   return moment.utc(trimmedDate, 'YYYY-MM-DD');
 }
+
+export function toMomentFromRailsDateTime(railsDateTimeString) {
```

Comment:
https://github.com/studentinsights/studentinsights/pull/1970 aims at simplifying this - this function was already here but was in another file.